pkg/apis/core/validation/installation.go

```diff
@@ -6,19 +6,41 @@ package validation
 
 import (
 	apivalidation "k8s.io/apimachinery/pkg/api/validation"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
+	"github.com/gardener/landscaper/pkg/apis/core/v1alpha1/helper"
+
 	"github.com/gardener/landscaper/pkg/apis/core"
 )
 
+const InstallationNameMaxLength = validation.DNS1123LabelMaxLength - len(helper.InstallationPrefix)
+
+const InstallationGenerateNameMaxLength = InstallationNameMaxLength - 5
```

Comment:
can you add a comment why the max installation name is substracted by 5.
Something like `max length of an installation - the number of random characters kubernetes uses to genreate a name`

Reply:
done